tests/docker/docker-compose.yml

```diff
@@ -42,5 +42,6 @@ services:
     - minio
     volumes:
     - '${BUILD_ROOT}:${BUILD_ROOT}'
+    - '${CLUSTERS_HOST_PATH:-/mnt/scratch}:${CLUSTERS_HOST_PATH:-/mnt/scratch}'
```

Comment:
where is `CLUSTERS_HOST_PATH` defined? I think this can be left as `/mnt:/mnt` . it's safe to do, as it always exists on linux boxes.

Reply:
fixed

Reply:
I reverted this change and hardcoded `/mnt/scratch` instead of `/mnt`. I was constantly seeing failures so I assume that something was breaking mounting the entire `/mnt`... to test my theory I pushed `/mnt/scratch:/mnt/scratch` and see how the build proceeds. `docker` created the dir `/mnt/scratch` on both host and container if it doesn't exist so `docker compose up` won't fail if `/mnt/scratch` doesn't exist on the host machine

Reply:
also, if this is hardcoding the path, then the use of a variable can be dropped, to avoid confusion

Reply:
This volume is no longer needed on the containers. With the latest changes introduced on the vtools PR, we don't need to update the docker-compose config. In fact the only change we now need on this PR is the changes in https://github.com/redpanda-data/redpanda/pull/4750/commits/49f659800b7dbe176f26be0ecf354b15ff2d30eb